dev-libs/stp/stp-2.3.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Simple Theorem Prover, an efficient SMT solver for bitvectors"
+HOMEPAGE="https://stp.github.io/"
+SRC_URI="https://github.com/stp/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="MIT GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="python static test"
+
+DEPEND="
+	sci-mathematics/minisat
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -i "s/set(LIBDIR lib/set(LIBDIR $(get_libdir)/" CMakeLists.txt
```

Comment:
Append `|| die` here please, 
https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html
To me this sedding is quite self-explanatory, but for everyone it might not be. Could you add a `# Short comment` above telling what it does?

Also `default` is usually the first thing called in src_prepare.

Reply:
They both do the same thing and they both work.